example/src/main/java/com/thefuntasty/mvvmsample/domain/LoginCompletabler.kt

```diff
@@ -9,19 +9,13 @@ import javax.inject.Inject
 
 class LoginCompletabler @Inject constructor(
     private val userStore: UserStore
-) : BaseCompletabler() {
+) : BaseCompletabler<LoginCompletabler.LoginData>() {
```

Comment:
This will be very ugly when the name of the interactor will be a very long
```
) : BaseCompletabler<RemoveCurrentlyInactiveDataFromBackendCompletabler.Data>() {
```
Wouldn't it be better to add these classes outside of the interactor? Below the interactor for example?